learning/basics/task5/solutions/PW-Palindrome_easy.py

```diff
@@ -0,0 +1,24 @@
+"""
+Palindrome is expression which looks the same whether you read it from the beginning or the end. Example of palindromic
+numbers: 121, 5445, 53835, 123454321
+
+Implement a function that will check if given number is palindrome. Function should return boolean value (True or False)
+"""
+palindromes = [12321, 543345, 9048409, 25452]
+not_palindromes = [123421, 1232, 987689]
+
+
+def is_palindrome(number):
+    number_list = []
+    for i in str(number):
+        number_list.append(int(i))
+    number_list_reverse = number_list[::-1]
+    return number_list == number_list_reverse
```

Comment:
Your solution will work fine, but there are shorter and cleaner ways to solve this, consider two ways below:
- You can just simply convert string to list by use `list()` function, without unnecessary `for` loop
```suggestion
def is_palindrome(number):
    number_list = list(str(number))
    number_list_reverse = number_list[::-1]
    return number_list == number_list_reverse
```
- or you can even not to convert string to list, reversing will work also for string
```suggestion
def is_palindrome(number):
    number_reverse = str(number)[::-1]
    return str(number) == number_reverse
```
If you have more question about one of this ways, feel free to ask :)

Reply:
All clear. Thanks for the hint :)